UPGRADE.md

```diff
@@ -1,5 +1,10 @@
 # Upgrade to 3.0
 
+## BC BREAK: Removed dbal:import CLI command
+
+The `dbal:import` CLI command has been removed since it only worked with PDO-based drivers by relying on a non-documented behavior of the extension, and it was impossible to make it work with other drivers.
+Please use other database client applications for import, e.g. `mysql < data.sql`.
```

Comment:
I'm going to be annoying and say that _including only MySQL looks biased_. :P Can we add alternatives that are also obvious?
```md

Please use other database client applications for import,
e.g. `mysql < data.sql`, `psql < data.sql`, `cat data.sql | sqlite3 my.db` or equivalent.
```

Reply:
I'll have to figure out the ways to import SQL dumps on all supported platforms, but in general, the proposal is legit. If you can add some more examples from the top of your head, please do.

BTW, isn't `cat data.sql | sqlite3 my.db` same as `sqlite3 my.db < data.sql` but with an extra process for `cat`?

Reply:
I don't think we need _all_ supported platforms, MySQL, MariaDB, PostgreSQL and SQLite is enough imho.

> isn't `cat data.sql | sqlite3 my.db` same as `sqlite3 my.db < data.sql`

Likely yes. :)
```
$ sqlite3 <<<'select 123'
123
```

Reply:
> I don't think we need _all_ supported platforms, MySQL, MariaDB, PostgreSQL and SQLite is enough imho.

Makes sense. I'll update the PR later.

> ```
> $ sqlite3 <<<'select 123'
> 123
> ```

Speaking of which, isn't it a proper replacement for `dbal:run-sql`? Yes, it should works across the platforms but in the entire `Tools` namespace we have only two commands remaining: `dbal:import`, `dbal:run-sql` and `dbal:reserved-words`. The latter, based on some recently discovered issues (e.g. https://github.com/doctrine/dbal/issues/3152) may become obsolete in 3.0 since on the one hand, keywords cannot be fully supported in DBAL, on the other ORM should just quote everything. The notion of which names are reserved shouldn't be relevant for an app developer.